karma.saucelabs.conf.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+/* eslint-disable */
```

Comment:
Other than the indentation of testName I can't see any reason we would have to disable eslint for this file